api/references/commands.md

```diff
@@ -268,6 +268,7 @@ let success = await commands.executeCommand('vscode.openFolder', uri);
 * _left_ - Left-hand side resource of the diff editor
 * _right_ - Right-hand side resource of the diff editor
 * _title_ - Human readable title for the diff editor
+* _options_ - (optional) Editor options. See vscode.TextDocumentShowOptions
```

Comment:
```suggestion
* _options_ - (optional) Either the column in which to open, or editor options (see vscode.TextDocumentShowOptions)
```

Reply:
Happy to apply this, but FTR when I tried passing a ViewColumn like `vscode.ViewColumn.One` I got an error, reading something like "expected a column but got 1" which is a little surprising, since vscode.ViewColumn is a numeric enum.

Reply:
In any case, I've checked the box to allow edits by maintainers, so feel free to edit into a form you like and merge.

Reply:
> Happy to apply this, but FTR when I tried passing a ViewColumn like vscode.ViewColumn.One I got an error, reading something like "expected a column but got 1" which is a little surprising, since vscode.ViewColumn is a numeric enum.

Here's an example from our [API tests](https://github.com/microsoft/vscode/blob/e9a1af2326c28b3bc0d40ab7abaed2ce24941e57/extensions/vscode-api-tests/src/singlefolder-tests/window.test.ts#L432C3-L432C126):

'''ts
await commands.executeCommand('vscode.diff', leftDiff, rightDiff, 'Diff', { viewColumn: ViewColumn.Four, preview: false });
'''

Reply:
Yeah, can confirm passing an options bag works. Passing just the view column as the fifth arg was where I got an error